frontend/src/pages/destructionlist/review/DestructionListReview.loader.ts

```diff
@@ -49,18 +60,15 @@ export const destructionListReviewLoader = loginRequired(
     }: ActionFunctionArgs): Promise<DestructionListReviewContext> => {
```

Comment:
We can optimize the call at line 68 by putting it into the Promise.All coming after it

Reply:
There is also a weird interaction happening, which I am unsure on how we'd be able to make cleanly, I think it's a UI loading issue.

Whenever we click on a checkbox the UI re-draws due to the loader call firing off again. This loader call takes nearly 400ms, so from the time we click the checkbox this happens:

- Checkbox is clicked
- Visually checkbox ticks off as soon as loader call starts (almost instantly)
- Nothing happens now, checkbox is still off until the loader call is finished (~400ms)
- Visually checkbox ticks on again after the loader is finished (~400ms)

I think this might cause some issues, especially if the API calls would take nearly a second (or more). Saying that, I am unsure how we can easily fix this right now but we need to keep this in mind

Reply:
Yes, your'e right this is a bit weird. I'm unsure how to fix this properly right now but lets keep this in mind. Maybe we need to investigate a way to implement optimistic rendering, however I'm reluctant to introduce (too many) state machines in the pages themself.